### PR TITLE
Feature/146/EditAccountStyling/Cat

### PR DESCRIPTION
**Pull request by cathleenjia**
## Style RA Dashboard

- styled edit account page
- actually edits account details the backend for both landlord and referral agency

### Related PRs

None

### Migrations

None

### Tests Performed, Edge Cases

None

### //TODO

None

### Screenshots

image

CC: @wilsonw926

### CODE REVIEW

File: app/javascript/components/EditAccount.jsx [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+    this.handleDestroy = this.handleDestroy.bind(this);
+  }
+
+  /* takes in two arrays (our array of field names and our array of values)
+   * and combines the corresponding field and value as a key value pair to be
+   * returned as a dictionary that will be sent in our JSON requests.
+   */
+  convertToDict() {
+    const user = this.state.user;
+    const keys = ["name", "address", "email", "phone", "password"];
```
Comment (wilsonw926) [inline app/javascript/components/EditAccount.jsx]: The bug was the name of the password field is actually "password" not "encrypted_password"

File: app/javascript/components/EditAccount.jsx [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+  handleEmailChange = (e) => {
+    this.setState({ email: e.target.value });
+  }
+
+  //api destroy
+  handleDestroy() {
+    let id = this.props.user.id;
+    let type = this.props.type;
+    var request = null;
+    if (this.props.type === "landlord") {
```
Comment (wilsonw926) [inline app/javascript/components/EditAccount.jsx]: Cleaned up this part of the code because we no longer need to take into account of property/tenant.
